packages/@best/frontend/server/index.ts

```diff
@@ -23,6 +24,7 @@ export const Frontend = (config: FrontendConfig): express.Application => {
 
     const app: express.Application = express()
 
+    app.use(crossOriginIsolation())
```

Comment:
@nolanlawson From what I can tell, this needs to be set [here](https://github.com/salesforce/best/blob/cc559d4f7b0cbaf77b5081b562f425ea6608b891/packages/%40best/runner-abstract/src/index.ts#L41-L42) as that seems to be the place that serves the page used in testing.

Reply:
@sf-v My mistake! Fixed now.